README.md

```diff
@@ -31,7 +31,7 @@ Note: In general, track parameters are only defined for a subset of the challeng
 
 Note: It is recommended to store any track parameters in a json file and pass them to Rally using '--track-params=./params-file.json'. 
 
-Follow is an example of a valid paramter json file:
+Following is an example of a valid paramters json file:
```

Comment:
Typo: `paramters` needs to be `parameters`

Reply:
Not sure how I missed that good eye.  Thanks, should be fixed now